app/views/partials/foyer/recap-situation.html

```diff
@@ -3,7 +3,7 @@ <h3 class="text-center">Votre situation</h3>
 <h4 class="section-header">Votre foyer</h4>
 <individu-block ng-repeat="individu in situation.individus" individu="individu"></individu-block>
 
-<div ng-if="situation.logement">
+<div id="logement-block" ng-if="situation.logement">
```

Comment:
We minimisez the amounts of added `id` to the CSS, but the recap is hard to navigate as is.